components/renku_data_services/platform/blueprints.py

```diff
@@ -68,3 +73,90 @@ async def _patch_singleton_configuration(
             )
 
         return "/platform/config", ["PATCH"], _patch_singleton_configuration
+
+
+@dataclass(kw_only=True)
+class PlatformUrlRedirectBP(CustomBlueprint):
```

Comment:
issue: We need to implement `DELETE`. Admins should be able to delete redirects. It will be likely rarely used but we should have it.

Reply:
I was intending to do this in a follow-up PR when the admin UI for managing redirects is implemented, but I can do it here if that is preferred.